### PR TITLE
Sword and spear straps

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Vali has the funny strap.
Normal melee weapons have strap now too.
## Why It's Good For The Game
Dropping your unga stick when you take a tumble is bad.
## Changelog
:cl:
balance: Machete and spear now have straps
/:cl:

### CODE REVIEW

**Comment by Tailiik** [comment]
What the belt harness doin?

**Comment by yyzsong** [comment]
synth buff pr of the year

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -302,6 +302,10 @@
 	icon_state = "spear"
 	item_state = "spear"
 
+/obj/item/weapon/twohanded/spear/tactical/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/strappable)
```
[the remark this answers is not in the document]
Comment (Lumipharon) [reply code/game/objects/items/weapons/twohanded.dm:307]: ? harvester doesn't have strappable. the vali MODULE has an analogue.

**Review by ivanmixo — APPROVED** [review]

**Comment by chizzy376** [comment]
just put a pistol lace and bayonet on the machete if you want high dmg nondrop

**Comment by Sirsnek15** [comment]
this can't be real...

**Pull request merged**